docs/sql-error-conditions.md

```diff
@@ -1148,7 +1154,7 @@ Please increase executor memory using the --executor-memory option or "`<config>
 
 [SQLSTATE: 42001](sql-error-conditions-sqlstates.html#class-42-syntax-error-or-access-rule-violation)
 
-Found an invalid expression encoder. Expects an instance of `ExpressionEncoder` but got `<encoderType>`. For more information consult '`<docroot>`/api/java/index.html?org/apache/spark/sql/Encoder.html'.
+Found an invalid expression encoder. Expects an instance of ExpressionEncoder but got `<encoderType>`. For more information consult '`<docroot>`/api/java/index.html?org/apache/spark/sql/Encoder.html'.
```

Comment:
Similar changes made by https://github.com/apache/spark/pull/45226. Could you rebase your PR on the recent master.

Reply:
@MaxGekk done